lib/rouge/lexers/opentype_feature_file.rb

```diff
@@ -41,17 +47,18 @@ def self.keywords
           push :featurename
         end
         # } <tag> ;
-        rule %r/(\})((?:\s)*)/ do
+        rule %r/(\})((?:\s))/ do
           groups Punctuation, Text
           push :featurename
         end
         # solve include( ../path)
-        rule %r/(include)/i, Keyword, :includepath
+        rule %r/(include)(?!_dflt)/i, Keyword, :includepath
```

Comment:
Could this be simplified to the following?

```suggestion
        rule %r/include\b/i, Keyword, :includepath
```

Reply:
Oh, definitely, that works.